### PR TITLE
Update to latest otel

### PR DESCRIPTION
**Pull request by trask**

### CODE REVIEW

**Comment by heyams** [comment]
should we update these to AAD branch too? switching between main and aad will cause otel-fork to be out of sync.

**Review by heyams — APPROVED** [review]

**Comment by trask** [comment]
> should we update these to AAD branch too? switching between main and aad will cause otel-fork to be out of sync.

👍 i'll merge `main` to `aad` after merging this

**Pull request merged**
